README.md

```diff
@@ -1,3 +1,77 @@
 # @comet/dev-process-manager
 
+## Installation
+
+```console
+$ npm install
+$ npm run build
```

Comment:
I would expect installation instructions here, how I can use dev-pm in a project. Not how to develop dev-pm itself (that would be a different section)

Reply:
This is how it works at the moment
When Alex published the npm package we can add the required instructions